src/tests/ftest/pool/create_all.py

```diff
@@ -0,0 +1,177 @@
+        :avocado: tags=pool_create_tests,create_all_two_pools
+        """
+
+        self.add_pool_qty(2, namespace="/run/pool/*", create=False)
+        self.pool[0].size.update("50%")
+        self.pool[1].size.update("100%")
+
+        self.log.info("Creating a pool with 50% of the available storage")
+        self.pool[0].create()
+        self.pool[0].get_info()
+        tiers_bytes = [self.pool[0].info.pi_space.ps_space.s_total]
+
+        self.log.info("Checking pool size")
+        delta = 1 << 20
+        self.check_available_bytes(tiers_bytes[0][0]*2, tiers_bytes[0][1]*2, delta, 0)
+        self.update_available_bytes()
+
+        self.log.info("Creating a pool with 100% of the available storage")
+        self.pool[1].create()
+        self.pool[1].get_info()
+        tiers_bytes.append(self.pool[1].info.pi_space.ps_space.s_total)
+
+        self.log.info("Checking pool size")
+        self.check_available_bytes(tiers_bytes[1][0], tiers_bytes[1][1], delta, 0)
+        self.update_available_bytes()
+        self.check_available_bytes(0, 0)
+
+    def test_recycle_pools(self):
+        """Test pool creation and destruction
+
+        Test Description:
+            Create a pool with all the capacity of all servers. Verify that the pool created
+            effectively used all the available storage. Destroy the pool and repeat these steps 100
+            times. For each iteration, check that the size of the created pool is always the same.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,large
+        :avocado: tags=pool
+        :avocado: tags=pool_create_tests,create_all_recycle_pools
+        """
+
+        scm_avail_bytes, smd_avail_bytes = self.get_available_bytes()
+        scm_delta_bytes = 1 << 20 # 1MiB
+
+        for index in range(10):
+            self.log.info(f"Creating pool {index} with 100% of the available storage")
+            self.add_pool_qty(1, namespace="/run/pool/*", create=False)
+            self.pool[0].size.update("100%")
+            self.pool[0].create()
+            self.assertEqual(self.pool[0].dmg.result.exit_status, 0,
+                    f"Pool {index} could not be created")
+
+            self.log.info(f"Checking size of pool {index}")
+            self.pool[0].get_info()
+            tier_bytes = self.pool[0].info.pi_space.ps_space.s_total
+            self.assertLessEqual(abs(scm_avail_bytes-tier_bytes[0]), scm_delta_bytes,
+                    f"Invalid SCM size: want={scm_avail_bytes}, got={tier_bytes[0]}")
+            self.assertEqual(smd_avail_bytes, tier_bytes[1],
+                    f"Invalid SMD size: want={smd_avail_bytes}, got={tier_bytes[1]}")
+
+            self.log.info(f"Destroying pool {index}")
+            self.pool[0].destroy()
+            self.assertEqual(self.pool[0].dmg.result.exit_status, 0,
+                    f"Pool {index} could not be destroyed")
+
+            self.log.info(f"Checking size of available storage at iteration {index}")
```

Comment:
(pylint-logging-fstring-interpolation) Use lazy % formatting in logging functions